library/alloc/src/collections/btree/map.rs

```diff
@@ -137,6 +137,7 @@ pub(super) const MIN_LEN: usize = node::MIN_LEN_AFTER_SPLIT;
 /// *stat += random_stat_buff();
 /// ```
 #[stable(feature = "rust1", since = "1.0.0")]
+#[doc(alias = "map")]
```

Comment:
Given that you're adding `map` to `BTreeMap`, do you want to add `set` to `BTreeSet`? Either way they should show up in the results though, so not really necessary.

Reply:
Good idea, thanks. Added.

> Either way they should show up in the results though, so not really necessary.

This will make it show up in the top, right now it's a long way down. ![image](https://user-images.githubusercontent.com/23638587/107890921-8e5fa400-6ee9-11eb-9e87-e4c175775165.png)